src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/CommunityAPIService.kt

```diff
@@ -5,23 +5,18 @@ import mu.KLogging
 import org.springframework.beans.factory.annotation.Value
 import org.springframework.context.ApplicationListener
 import org.springframework.stereotype.Service
-import org.springframework.web.reactive.function.client.WebClient
 import org.springframework.web.util.UriComponentsBuilder
-import reactor.core.publisher.Mono
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.component.CommunityAPIClient
 import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.events.ReferralEvent
 import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.events.ReferralEventType
 import java.time.LocalDate
 
 @Service
 class CommunityAPIService(
-  @Value("\${community-api.contact-notification-context.provider-code}") private val providerCode: String,
-  @Value("\${community-api.contact-notification-context.referral-type}") private val referralType: String,
-  @Value("\${community-api.contact-notification-context.staff-code}") private val staffCode: String,
-  @Value("\${community-api.contact-notification-context.team-code}") private val teamCode: String,
```

Comment:
🙋 These are also mentioned in the configuration and helm values files -- can we remove them from those?

Reply:
Thanks for pointing that out. Searched the code base/git for provider-code - no refs. Where are they located?

Reply:
helm_deploy/hmpps-interventions-service/templates/_envs.tpl
helm_deploy/values-dev.yaml

They are referenced such as: COMMUNITYAPI_CONTACTNOTIFICATIONCONTEXT_TEAMCODE

Reply:
Found it and done thanks